packages/rexray/buildinfo.json

```diff
@@ -0,0 +1,9 @@
+{
+  "docker": "golang:1.6",
+  "single_source" : {
+    "kind": "git",
+    "git": "git@github.com:dcos/rexray.git",
+    "ref": "1816ee3868550e07b90f00f57bfd7f737235cd93",
+    "ref_origin": "v0.3.3-dcos"
```

Comment:
Ok thanks. @branden dvdcli artifact is now available at: https://downloads.dcos.io/pkgpanda-artifact-cache/dvdcli-Linux-x86_64-0.2.0.tar.gz